websocket/sessionImpl.go

```diff
@@ -0,0 +1,20 @@
+package websocket
```

Comment:
why did you name this file `sessionImpl.go`? What does impl mean? Can we at least use snake-case please.

Reply:
It supposes to be the session implementation file for incoming messages!